.cbmc-batch/jobs/aws_array_eq_c_str/Makefile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########
+include ../Makefile.aws_byte_buf
+
+UNWINDSET += aws_array_eq_c_str.0:$(shell echo $$(($(MAX_BUFFER_SIZE) + 1)))
```

Comment:
Wanna write an "add one" makefile macro? Would make our lives easier.

Reply:
Yes :)

Reply:
I'll address this on a separate PR, so I can update all `UNWINDSET` as well.